haskell-utils.el

```diff
@@ -100,12 +100,13 @@ execusion."
     (remove-hook
      'post-command-hook #'haskell-utils-async-update-post-command-flag t)))
```

Comment:
@gracjan I've spent few hour trying to make it record changes to flag during ert running test.  I have tried `command-execute`, `call-interactively`, `execute-kbd-macro`, and changing modes but flag remains empty all the time.  Can you help me with this?